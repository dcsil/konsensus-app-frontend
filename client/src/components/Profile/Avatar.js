import React from 'react';

const Avatar = ({ name }) => {
  console.log(name);
  return (
    <img
      className="rounded-circle"
      src={`https://ui-avatars.com/api/?size=128&font-size=0.36&background=random&rounded=true&name=${name}`}
      alt="..."
    />
  );
};

export default Avatar;
