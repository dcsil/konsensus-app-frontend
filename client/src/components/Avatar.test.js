import React from 'react';
import { shallow } from 'enzyme';
import Avatar from './Profile/Avatar';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

test('Test Sidebar component', () => {
  const wrapper = shallow(<Avatar />);
  expect(wrapper).toMatchSnapshot();
});
