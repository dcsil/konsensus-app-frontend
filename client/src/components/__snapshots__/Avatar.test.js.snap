// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Avatar component 1`] = `ShallowWrapper {}`;
